overlord/configstate/configcore/kernel.go

```diff
@@ -184,6 +185,18 @@ func isDangerousModel(st *state.State) (bool, error) {
 	return devCtx.Model().Grade() == asserts.ModelDangerous, nil
 }
 
+func isSeeded(st *state.State) (bool, error) {
```

Comment:
there are already helpers like this in the package, maybe time to clean this up?  I see a check for seeded in cloud.go and netplan.go

Reply:
I've reused the code now